core/src/main/resources/hudson/model/Messages_fr.properties

```
@@ -157,28 +342,69 @@ ChoiceParameterDefinition.MissingChoices=Choix requis.
 
-LoadStatistics.Legends.TotalExecutors=Nombre total d''exécuteurs
+LoadStatistics.Legends.DefinedExecutors=Éxecuteurs déclarés
 LoadStatistics.Legends.OnlineExecutors=Nombre d''exécuteurs en ligne
+LoadStatistics.Legends.ConnectingExecutors=Nombre d''éxecuteurs en cours de connexion
+LoadStatistics.Legends.TotalExecutors=Nombre total d''exécuteurs
 LoadStatistics.Legends.BusyExecutors=Nombre d''exécuteurs occupés
+LoadStatistics.Legends.IdleExecutors=Nombre d''exécuteurs en attente
 LoadStatistics.Legends.AvailableExecutors=Nombre d''exécuteurs disponibles
 LoadStatistics.Legends.QueueLength=Longueur de la file d''attente
 
 Cause.LegacyCodeCause.ShortDescription=Ce job a été lancé par du code legacy. Pas d''information sur les causes
 Cause.UpstreamCause.ShortDescription=Démarré par le projet amont "{0}" de numéro de build {1}
+Cause.UpstreamCause.CausedBy=Causé à l''origine par :
 Cause.UserCause.ShortDescription=Démarré par l''utilisateur {0}
 Cause.UserIdCause.ShortDescription=Démarré par l''utilisateur {0}
-Cause.RemoteCause.ShortDescription=Démarré à distance par {0}
+Cause.RemoteCause.ShortDescription=Démarré à distance par l''hôte {0}
+Cause.RemoteCause.ShortDescriptionWithNote=Demarré à distance par l''hôte {0} avec le commentaire : {1}
+
+ProxyView.NoSuchViewExists=La vue globale {0} n''existe pas
+ProxyView.DisplayName=Inclure une vue globale
 
 MyViewsProperty.DisplayName=Mes vues
 MyViewsProperty.GlobalAction.DisplayName=Mes vues
+MyViewsProperty.ViewExistsCheck.NotExist=La vue avec le nom {0} n''existe pas
+MyViewsProperty.ViewExistsCheck.AlreadyExists=La vue avec le nom {0} existe déjà
 
-ManageJenkinsAction.DisplayName=Administrer Jenkins
-ParametersDefinitionProperty.DisplayName=Ce build a des paramètres
+CLI.restart.shortDescription=Redémarrer Jenkins
+CLI.safe-restart.shortDescription=Redémarrer Jenkins en sûreté
+CLI.keep-build.shortDescription=Conserver cette construction sans limite de temps.
+
+BuildAuthorizationToken.InvalidTokenProvided=Le token fourni est invalide.
+
+Jenkins.CheckDisplayName.NameNotUniqueWarning=Le nom affiché, "{0}", est utilisé comme nom de job et peut causer des résultats de recherche erronés.
+Jenkins.CheckDisplayName.DisplayNameNotUniqueWarning=Le nom affiché, "{0}", est déjà utilisé par un autre job et peut engender des erreurs et des délais.
+
+Jenkins.NotAllowedName=“{0}” n''est pas un nom autorisé
+Jenkins.IsRestarting=Jenkins est en cours de redémarrage
+
+User.IllegalUsername="{0}" est interdit comme nom d''utilisateur pour des raisons de sécurité.
+User.IllegalFullname="{0}" est interdit comme nom complet pour des raisons de sécurité.
+
+TimeZoneProperty.DisplayName=Fuseau horaire défini par l''utilisateur
+TimeZoneProperty.DisplayDefaultTimeZone=Défaut
+TimeZoneProperty.current_time_in_=Heure actuelle dans le fuseau horaire {0} : {1}
+TimeZoneProperty.current_time_on_server_in_in_proposed_di=Heure actuelle sur le serveur dans le fuseau horaire {0} : {1}; dans le fuseau horaire : {2}
```

```suggestion
TimeZoneProperty.current_time_on_server_in_in_proposed_di=Heure actuelle sur le serveur dans le fuseau horaire {0} : {1}; dans le fuseau horaire proposé : {2}
```
Required missing "proposé"